RELEASE.md

```diff
@@ -31,17 +31,45 @@ For you to follow along according to these instructions, you need:
    git clean -xfd
    ```
 
-1. Update the version with `bump2version` (can be installed with `pip install -r dev-requirements.txt`)
+1. Set the `version_info` variable in [\_version.py](_version.py) appropriately
+   and make a commit.
 
-   ```bash
+   ```
+   git add _version.py
    VERSION=...  # e.g. 1.2.3
-   bump2version --tag --new-version $VERSION -
+   git commit -m "release $VERSION"
    ```
 
-1. Reset the version to the next development version with `bump2version`
+1. Reset the `version_info` variable in
+   [\_version.py](_version.py) appropriately with an incremented
+   patch version and a `dev` element, then make a commit.
 
-   ```bash
-   bump2version --no-tag patch
+   ```
+   git add _version.py
+   git commit -m "back to dev"
+   ```
+
+1. Push your two commits to master.
```

Comment:
Change to `...commits to main`. There are other references to `master` so will ignore the rest.